websqrl/src/EventDomain.ts

```diff
@@ -0,0 +1,145 @@
+import { getFileNameForDate } from "./getFileNameForDate";
+import type { EventData } from "./types";
+
+// TODO(meyer) actual jsonpath stuff
```

Comment:
I don't think we actually need jsonPath, should be safe to assume the date is in a top level field.

I'd probably just rename this to `getDateFromField` so it's doing what it says, and that's good enough for now?

Reply:
good idea, can do